python/cudf/cudf/core/indexed_frame.py

```diff
@@ -1961,6 +1961,11 @@ def drop_duplicates(
         ignore_index: bool, default False
             If True, the resulting axis will be labeled 0, 1, ..., n - 1.
         """
+        if not isinstance(ignore_index, bool):
```

Comment:
Does pandas allow `np.bool_`'s too? If yes, we will need to allow that too.

Reply:
Good catch. Yes pandas accepts `np.bool_` so added that here

Reply:
Thanks @mroeschke !